property-service/src/main/java/org/egov/ps/service/EnrichmentService.java

```diff
@@ -294,6 +297,44 @@ public void enrichCreateApplication(ApplicationRequest request) {
 			setIdgenIds(request);
 		}
 	}
+	
+	public void enrichProcessNotifications(ApplicationRequest request) {
+		// TODO Auto-generated method stub
+		// call processNotification method
+
+		request.getApplications().forEach(application -> {
+
+			List<Map<String, Object>> notificationConfigs = mdmsservice.getNotificationConfig("notifications", request.getRequestInfo(), application.getTenantId(), application);
+			processNotification(notificationConfigs, application);
+
+		});
+
+	}
+	
+	public void processNotification(List<Map<String, Object>> notificationConfigs, Application enrichedApplication) {
+		if (!CollectionUtils.isEmpty(notificationConfigs)) {
+			ObjectMapper mapper = new ObjectMapper();
+			List<Notifications> notificationList = mapper.convertValue(notificationConfigs, new TypeReference<List<Notifications>>() { });
+
+			notificationList.forEach(data -> {
+				data.setContent(data.getContent()
+						//app.getModuleType() == null ? "" : app.getModuleType()
+						.replace("{createdBy.name}",enrichedApplication.getAuditDetails() == null ? "" : enrichedApplication.getAuditDetails().getCreatedBy())
```

Comment:
we are still hardcoding `{createdBy.name}` instead of extracting value using JsonPath.

Reply:
@rajiv-transerve , as i did code i cannot use json path ... last you explain but way of code i think this is correct one